NP/Unicode.hs

```diff
@@ -57,6 +57,48 @@ greekData = [(Just 'a', "alpha", "α")
             ,(Just 'O', "Omega", "Ω")
             ]
 
+-- | Elder Futhark runes
+futhark :: [(String, String)]
+futhark = [(name, unicode) | (_,name,unicode) <- futharkData] ++
+        [ ([leading,shorthand],unicode)
+        | (Just shorthand,_,unicode) <- futharkData
+        , leading                    <- ['\'', 'f'] ]
```

Comment:
Marking `'` as a leading character will conflict with all the greek letters.
However, I'm fine with the `f` as a leading character as it does not conflict directly.
Symbols starting with `f` are: `forall` (∀) and `fort` (∜), this means that `fo` will only produce `othlia` (ᛟ) with a space afterward.